pkg/util/operatorconfig/modules.go

```diff
@@ -0,0 +1,59 @@
+package operatorconfig
+
+import (
+	"encoding/json"
+	"os"
+	"sync"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/logd"
+)
+
+const (
+	modulesJsonEnv = "modules.json"
+)
+
+var (
+	once sync.Once
+
+	modules Modules
+
+	fallbackModules = Modules{
+		ActiveGate:  true,
+		OneAgent:    true,
+		Extensions:  true,
+		LogModule:   true,
+		EdgeConnect: true,
+	}
+
+	log = logd.Get().WithName("operator-config")
+)
+
+type Modules struct {
+	ActiveGate  bool `json:"activeGate"`
+	OneAgent    bool `json:"oneAgent"`
+	Extensions  bool `json:"extensions"`
+	LogModule   bool `json:"logModule"`
+	EdgeConnect bool `json:"edgeConnect"`
+}
+
+func GetModules() Modules {
+	once.Do(func() {
+		modulesJson := os.Getenv(modulesJsonEnv)
+		if modulesJson == "" {
+			log.Info("operator config envvar not set, using default", "envvar", modulesJsonEnv)
```

Comment:
```suggestion
			log.Info("operator-config envvar not set, using default", "envvar", modulesJsonEnv)
```

Reply:
just for consistency, as you used a "-" in the next log message. (i know way too nitpicky)

Reply:
made it nicer